src/libasr/codegen/wasm_to_x64.cpp

```diff
@@ -21,7 +21,7 @@ pushed back onto the stack.
 
 One of the reasons to use stack to pass function arguments is that,
 it allows us to define and call functions with any number of parameters.
-As registers are limited in number, if we use them to pass function arguments,
+As registers are limited in number, if we use them to pass function arugments,
```

Comment:
I think some of the WASM stuff should be reverted, I think @Shaikh-Ubaid mostly worked in LPython.